clar2wasm/benches/comparison.rs

```diff
@@ -0,0 +1,348 @@
+#![allow(clippy::expect_used, clippy::unwrap_used)]
+
+use clar2wasm::compile;
+use clar2wasm::datastore::{BurnDatastore, Datastore, StacksConstants};
+use clar2wasm::initialize::initialize_contract;
+use clarity::consts::CHAIN_ID_TESTNET;
+use clarity::types::StacksEpochId;
+use clarity::vm::analysis::{run_analysis, AnalysisDatabase};
+use clarity::vm::ast::build_ast_with_diagnostics;
+use clarity::vm::contexts::GlobalContext;
+use clarity::vm::costs::LimitedCostTracker;
+use clarity::vm::database::{ClarityDatabase, MemoryBackingStore};
+use clarity::vm::types::{
+    BuffData, QualifiedContractIdentifier, SequenceData, StandardPrincipalData,
+};
+use clarity::vm::{
+    eval_all, CallStack, ClarityVersion, ContractContext, ContractName, Environment, Value,
+};
+use criterion::{criterion_group, criterion_main, Criterion};
+use paste::paste;
+use pprof::criterion::{Output, PProfProfiler};
+
+#[inline(always)]
```

Comment:
Why do we need this? What kind of changes does it make on the generated assembly?

Reply:
It makes sure that there's only a single codepath per benchmark entry. In this case I don't think it leads to anything but a bloated binary in the end. Will remove.